php/plan.sh

```diff
@@ -39,27 +41,27 @@ pkg_include_dirs=(include)
 pkg_interpreters=(bin/php)
 
 do_build() {
+  attach
```

Comment:
Hi @predominant.  This is pausing the build.  Can you please remove?

Reply:
Thanks @gavindidrichsen - Fixed.